excelerator/imports/api/methods-init.js

```diff
@@ -225,16 +235,43 @@ WHERE {
                     datasetUri: "http://linked.data.gov.au/dataset/gnaf",
                     title: "Street Locality",
                     uri: "http://linked.data.gov.au/def/gnaf#StreetLocality",
+                    prefix: "http://linked.data.gov.au/dataset/gnaf/streetLocality/",
                     withinTypes: [
                         "http://linked.data.gov.au/def/gnaf#Locality"
                     ],
                 }, {
                     datasetUri: "http://linked.data.gov.au/dataset/gnaf",
                     title: "Locality",
                     uri: "http://linked.data.gov.au/def/gnaf#Locality",
+                    prefix: "http://linked.data.gov.au/dataset/gnaf/locality/"
+                }];
+
+                var gnaf16 = [{
+                    datasetUri: "http://linked.data.gov.au/dataset/gnaf-2016-05",
+                    title: "Address",
+                    uri: "http://linked.data.gov.au/def/gnaf#Address",
+                    prefix: "http://linked.data.gov.au/dataset/gnaf-2016-05/address/",
+                    withinTypes: [
+                        "http://linked.data.gov.au/def/gnaf#StreetLocality",
+                        "http://linked.data.gov.au/def/gnaf#Locality"
+                    ],
+                    baseType: true,
+                }, {
+                    datasetUri: "http://linked.data.gov.au/dataset/gnaf-2016-05",
+                    title: "Street Locality",
+                    uri: "http://linked.data.gov.au/def/gnaf#StreetLocality",
+                    prefix: "http://linked.data.gov.au/dataset/gnaf-2016-05/streetLocality/",
+                    withinTypes: [
+                        "http://linked.data.gov.au/def/gnaf#Locality"
+                    ],
+                }, {
+                    datasetUri: "http://linked.data.gov.au/dataset/gnaf-2016-05",
+                    title: "Locality",
+                    uri: "http://linked.data.gov.au/def/gnaf#Locality",
+                    prefix: "http://linked.data.gov.au/dataset/gnaf-2016-05/locality/"
```

Comment:
Not a big fan of the copy here. It would have been better to modify the mapping function below to add the prefix, as most of the other properties are the same. Something like:
```
var gnaf16 = gnafCurrent.map(x => Object.assign({}, x, { 
    datasetUri: "http://linked.data.gov.au/dataset/gnaf-2016-05",
    prefix: `http://linked.data.gov.au/dataset/gnaf-2016-05/${x.split('/').slice(-1)[0]}`
 }));
```

Its also possiblem while we are building in assumptions, that the prefix should just be the extension path ontop of the datasetUri. That would also reduce duplication.

Reply:
Thanks - probably better to use a mapping function rather than a copy.  

Yes, datasetURI could prepend the prefix. I can look at making these changes.

Reply:
done these now